index.html

```diff
@@ -1,12 +1,311 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header2">
+      <a href="#">Welcome</a>
+      <nav>
+        <img class="menu-button" src="./images/Enabled.png" alt="mobile-menu" />
+      </nav>
+    </header>
```

Comment:
This is a nice header. Next time please do ensure to enclose your nav tag in a section tag.
```suggestion
<section>
    <header class="header2">
      <a href="#">Welcome</a>
      <nav>
        <img class="menu-button" src="./images/Enabled.png" alt="mobile-menu" />
      </nav>
    </header>
    </section>
```